### PR TITLE
Updated travis ci for gradle 5 support

### PR DESCRIPTION
**Pull request by gianluz**
fixes https://github.com/danger/kotlin/issues/16

### CODE REVIEW

**Comment by f-meloni** [comment]
Should you update `kscript` as well to work with gradle 5?

File: .travis.yml [hunk 1]
```diff
@@ -27,3 +27,29 @@ matrix:
       script:
         - make install
         - danger-kotlin ci
+    - os: osx
+      osx_image: xcode10
+      install:
+        - curl -s "https://get.sdkman.io" | bash
+        - source ~/.bash_profile
+        - sdk install kscript
+        - sdk install gradle 5.2.1
```
Comment (f-meloni) [inline .travis.yml]: If you remove the version it **should** take the latest one, that is the one we most care about probably if we take the brew way

Comment (gianluz) [reply .travis.yml]: i always use to use fixed versions, if there is an updated version could break something and we not have any control of that. I think is a good idea use known versions. BTW, with gradle 5 there is some problems on kotlin native.. i'm working on that

Comment (f-meloni) [reply .travis.yml]: You are right, but in this specific case, given we are going to use brew as explained in the article @orta posted here https://github.com/danger/kotlin/issues/13#issuecomment-460879054, brew doesn't use versions, this means that if we add gradle as dependency who installs danger-kotlin gets it with the latest version available, then I would suggest to test against the one the people will get

File: build.gradle [hunk 2]
```diff
@@ -1,5 +1,19 @@
 buildscript {
     repositories {
         mavenCentral()
+        maven { url 'https://dl.bintray.com/kotlin/kotlin-dev' }
```
Comment (f-meloni) [inline build.gradle]: Why do you need two with dev and eap?

File: danger-kotlin-library/build.gradle [hunk 3]
```diff
@@ -1,5 +1,5 @@
 plugins {
-    id 'org.jetbrains.kotlin.jvm' version '1.3.0'
+    id 'org.jetbrains.kotlin.jvm'
```
Comment (f-meloni) [inline danger-kotlin-library/build.gradle]: No fixed version?

File: danger-kotlin/build.gradle [hunk 4]
```diff
@@ -1,17 +1,17 @@
 plugins {
-    id 'org.jetbrains.kotlin.konan' version "1.3.11"
+    id 'kotlin-multiplatform'
```
Comment (f-meloni) [inline danger-kotlin/build.gradle]: 🎉

File: danger-kotlin/src/runnerMain/kotlin/com/danger/runner/DangerKotlin.kt [hunk 5]
```diff
@@ -1,5 +1,8 @@
+package com.danger.runner
+
 import platform.posix.*
 
+
```
Comment (f-meloni) [inline danger-kotlin/src/runnerMain/kotlin/com/danger/runner/DangerKotlin.kt]: not needed space

File: danger-kotlin-library/build.gradle (hunk 3 above)
Comment (gianluz) [reply danger-kotlin-library/build.gradle]: not needed.. is defined in gradle properties

**Pull request merged**